apiclient/client.go

```diff
@@ -63,6 +67,9 @@ func NewHTTPclient(addr *url.URL, bearerToken *uuid.UUID) (*HTTPclient, error) {
 		return nil, fmt.Errorf("cannot get chain ID from API server")
 	}
 	c.chainID = info.ID
+
+	// Get the default circuit config
+	c.circuit = circuit.CircuitsConfigurations[info.CircuitConfigurationTag]
```

Comment:
Since this is a map, you should receive the second parameter `ok` and check if its true (else raise an error)

Reply:
Solved!